package.json

```diff
@@ -15,7 +15,7 @@
     "startclient": "node ./src/comm/client/zoo-client.js",
     "burrow-deps": "npm install --no-save @monax/burrow@0.23.0 grpc@1.16.1",
     "composer-v0.19-deps": "npm install --no-save composer-admin@0.19.18 composer-client@0.19.18 composer-common@0.19.18 fabric-ca-client@1.1.0 fabric-client@1.1.0",
-    "composer-deps": "npm install --no-save composer-admin@0.19.18 composer-client@0.19.18 composer-common@0.19.18 fabric-ca-client@1.2.0 fabric-client@1.2.0",
+    "composer-deps": "npm install --no-save composer-admin@0.19.18 composer-client@0.19.18 composer-common@0.19.18 fabric-ca-client@1.1.0 fabric-client@1.1.0",
```

Comment:
Now it's the same as `composer-v0.19-deps`, so line 17 can be removed

Reply:
Fixed, please take a look.